src/pages/components/callout-with-media.mdx

```diff
@@ -23,9 +24,19 @@ Callout &mdash; with media uses high contrast colors to create a delightful paus
 
 ![Default](../../images/pattern/callout-with-media/callout-with-media.jpg)
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ----------|
+| Headline | Short copy | Yes   | 1 | XXS (40 / 55)|
+| Paragraph | Long copy | No  | 1 | XXL (350 / 450) |
+| [Video player](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/video-player) or [Image with caption](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/image-with-caption) | Component | Yes | 1 | - |
+
+To see the list of all character count standards, go to this <a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer noopener nofollow">link</a>.
```

Comment:
I added a comment to a previous PR around this. I'll stop commenting on it for this round, and assume you'll take a look across the aboard at all your PRs.

Reply:
👍